src/smart-components/role/add-role-new/add-permissions-template.js

```diff
@@ -47,6 +49,16 @@ const AddPermissionTemplate = ({ formFields }) => {
           )}
         </Text>
       </TextContent>
+      {notAllowedBasePermissions?.length > 0 && !alertClosed ? (
```

Comment:
You should wrap these in parenthesis.
```suggestion
      {(notAllowedBasePermissions?.length > 0 && !alertClosed) ? (
```

Reply:
@karelhala ESLint does not allow me to do it